From 9602142d53f08f13a5da6e9ec3f724029791c36b Mon Sep 17 00:00:00 2001
From: Jaxson Han <jaxson.han@arm.com>
Date: Tue, 16 May 2023 10:32:37 +0800
Subject: [PATCH] arch: arm64: mpu: Remove LOG print before mpu enabled

LOG system has unalignment access instruction which will cause an
alignment exception before MPU is enabled. Remove the LOG print before
MPU is enabled to avoid this issue.

Signed-off-by: Jaxson Han <jaxson.han@arm.com>
Upstream-Status: Backport [main: 101ae5d2406b567e835a9641bbbf380bcc4a154e]
---
 arch/arm64/core/cortex_r/arm_mpu.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/core/cortex_r/arm_mpu.c b/arch/arm64/core/cortex_r/arm_mpu.c
index 58b7dcd1ebbc..5407e1f8c569 100644
--- a/arch/arm64/core/cortex_r/arm_mpu.c
+++ b/arch/arm64/core/cortex_r/arm_mpu.c
@@ -180,8 +180,6 @@ void z_arm64_mm_init(bool is_primary_core)
 		return;
 	}
 
-	LOG_DBG("total region count: %d", get_num_regions());
-
 	arm_core_mpu_disable();
 
 	/* Architecture-specific configuration */
-- 
2.34.1

